torch/_torch_docs.py

```diff
@@ -2220,13 +2220,14 @@ def parse_kwargs(desc):
            r"""
 kthvalue(input, k, dim=None, keepdim=False, out=None) -> (Tensor, LongTensor)
 
-Returns the :attr:`k` th smallest element of the given :attr:`input` tensor
+Returns the  of the given :attr:`input` tensor
```

Comment:
Removed the comment by mistake?

Reply:
@fmassa Yes, you are right, it's a mistake. I actually mean to remove this whole paragraph in favor of the next paragraph. But I only removed these few words by mistake.